lib/replicator.js

```diff
@@ -1843,10 +1873,8 @@ module.exports = class Replicator {
     // Also auto compresses the range based on local bitfield
     clampRange(this.core, r)
 
-    this._ranges.push(r)
```

Comment:
Was this accidental? This is the only place where we add a range to `this._ranges`.